lunatrace/bsl/ingest-worker/pkg/vulnerability/advisory/osv.go

```diff
@@ -30,6 +29,26 @@ func isReviewed(rawVuln *schema.OsvSchema) bool {
 	return rawVuln.DatabaseSpecific["github_reviewed"] == true
 }
 
+// ('Critical','High','Medium','Low','Negligible','Unknown')
+func getSeverityName(rawVuln *schema.OsvSchema) string {
```

Comment:
this seems to not work reliably, I think something is janky with the upsert. SOME of them are getting updated, but not all. I'd like to wipe out my DB and see if that fixes the issue, but since we wont have that advantage in production, I'm not sure what the trouble is.

Reply:
@ajvpot any ideas why this only upserts a fraction of the vulnerabilities with the proper column? Anything jump out at you?

Reply:
do you have any example vulnerabilities you expect to get upserted, but are not?